tests/Kernel/ConsultationRenderTest.php

```diff
@@ -30,6 +30,7 @@ class ConsultationRenderTest extends ContentRenderTestBase {
     'oe_time_caching',
     'oe_content_departments_field',
     'oe_content_consultation',
+    'oe_content_sub_entity_document_reference',
```

Comment:
Please test do we need to define oe_content_sub_entity_document_reference here?

Reply:
It's needed in kernel tests otherwise the entity type is missing.